plugin-api/build.gradle

```diff
@@ -71,7 +71,7 @@ Calculated : ${currentHash}
 tasks.register('checkAPIChanges', FileStateChecker) {
   description = "Checks that the API for the Plugin-API project does not change without deliberate thought"
   files = sourceSets.main.allJava.files
-  knownHash = '4jVaj9yW88nHbX0KmTR3dPQRvj9x8Pvh5E9Ry7KRT6w='
+  knownHash = 'VN2JB2HPpEUDQaDvd7QcMkmmgedasVChfA8tnSf1GHU='
```

Comment:
what has changed in the plugin API?

Reply:
Added a method to LabelledGauge to allow us to check whether a label is observed or not

Reply:
Can we add a default method (returns a dummy false value?) to avoid the breaking change please?

Reply:
Yep, good idea